src/rez/utils/_version.py

```diff
@@ -4,4 +4,4 @@
 
 # Update this value to version up Rez. Do not place anything else in this file.
 # Using .devN allows us to run becnmarks and create proper benchmark reports on PRs.
-_rez_version = "3.1.1"
+_rez_version = "3.1.2"
```

Comment:
```suggestion
_rez_version = "3.1.1"
```
We only update the version when we are ready to release. We will take care of that.